data-model/fdo-type/machine-annotation-service/0.3.0/examples/mas-example.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@id": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+  "@type": "ods:MachineAnnotationService",
+  "ods:ID": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+  "ods:type": "https://doi.org/10.15468/1a2b3c",
+  "schema:version": 1,
+  "schema:name": "GBIF Linker Service",
+  "schema:description": "A service that links GBIF records to a DiSSCo Digital Specimen. It creates an EntityRelationship indicating the relationship",
+  "schema:dateCreated": "2021-06-01T12:00:01Z",
+  "ods:dateTombstoned": "2023-01-01T12:00:01Z",
+  "ods:creator": "e2befba6-9324-4bb4-9f41-d7dfae4a44b0",
+  "ods:containerImage": "public.ecr.aws/dissco/mindat-georeferencing",
+  "ods:containerTag": "sha-cb76994",
+  "ods:TargetDigitalObjectFilter": {
+    "$.ods:type": [
+      "https://doi.org/21.T11148/894b1e6cad57e921764e"
+    ],
+    "$.ods:topicDiscipline": [
+      "Astrogeology",
+      "Geology",
+      "Palaeontology",
+      "Other Geodiversity",
+      "Unclassified"
+    ],
+    "$.occurrences[*].location.dwc:locality": [
```

Comment:
Same here, occurrences does not exist anymore right

Reply:
Fixed